packages/common/src/components/Footer.tsx

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import styled from 'styled-components';
+import MaxWidth from './flex/MaxWidth';
+
+import media from '../utils/media';
+
+const Container = styled.div`
+  display: flex;
+  justify-content: space-around;
+  width: 100%;
+  padding-top: 5rem;
+  padding-bottom: 3rem;
+  flex-wrap: wrap;
+`;
+
+const Column = styled.div`
+  width: calc(33% - 2rem);
+  margin: 0 1rem;
+
+  ${media.phone`
+    width: 100%;
+    margin-bottom: 1rem;
+  `};
+`;
+
+const Title = styled.h5`
+  font-size: 1.125rem;
+  font-weight: 400;
+  margin: 0;
+  margin-bottom: 1rem;
+
+  color: ${({ theme }) => theme.secondary};
+`;
+
+const List = styled.ul`
+  color: rgba(255, 255, 255, 0.7);
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+
+  li {
+    a {
+      transition: 0.3s ease color;
+      text-decoration: none;
+      color: rgba(255, 255, 255, 0.7);
+
+      &:hover {
+        color: rgba(255, 255, 255, 0.9);
+      }
+    }
+  }
+`;
+
+const Authors = styled.div`
+  color: white;
+  font-size: 0.875rem;
+  text-align: right;
+`;
+
+const Background = styled.div`
+  position: relative;
+  background-color: ${props => props.theme.background2.darken(0.2)};
+  padding: 1rem;
+  z-index: 5;
+`;
+
+const BasComponent = () => (
+  <a
+    id="bas"
+    href="https://www.aedin.com/in/basbuursma/"
```

Comment:
Did you replace `link` with `a`? 😂

Reply:
Also don't know why bas has it's own component ahah